pkg/controller/master/node/node_down_controller.go

```diff
@@ -3,278 +3,239 @@ package node
 import (
 	"context"
 	"fmt"
+	"reflect"
+	"slices"
 	"time"
 
-	longhorntypes "github.com/longhorn/longhorn-manager/types"
 	ctlcorev1 "github.com/rancher/wrangler/v3/pkg/generated/controllers/core/v1"
 	"github.com/sirupsen/logrus"
-	"go.uber.org/multierr"
 	corev1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/labels"
-	"k8s.io/utils/set"
-	kubevirtv1 "kubevirt.io/api/core/v1"
 
-	harvesterv1 "github.com/harvester/harvester/pkg/apis/harvesterhci.io/v1beta1"
 	"github.com/harvester/harvester/pkg/config"
-	v1 "github.com/harvester/harvester/pkg/generated/controllers/kubevirt.io/v1"
-	ctlstoragev1 "github.com/harvester/harvester/pkg/generated/controllers/storage.k8s.io/v1"
 	"github.com/harvester/harvester/pkg/settings"
 )
 
 const (
 	nodeDownControllerName = "node-down-controller"
+	kubevirtDrainTaintKey  = "kubevirt.io/drain"
+	nonGracefulTaintKey    = "node.kubernetes.io/out-of-service"
```

Comment:
can we use the `corev1.TaintNodeOutOfService` as the value, instead of the literal?